spec/factories/steps.rb

```diff
@@ -2,8 +2,14 @@
 
 FactoryGirl.define do
   factory :step do
-    message_type
-    time_stamp { Time.now }
-    command
+    message_type "single_command"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.